code/modules/modular_computers/hardware/hardware.dm

```diff
@@ -6,7 +6,7 @@
 	var/power_usage = 0					// If the hardware uses extra power, change this.
 	var/enabled = TRUE					// If the hardware is turned off set this to 0.
 	var/critical = TRUE					// Prevent disabling for important component, like the HDD.
-	var/hardware_size = 1				// Limits which devices can contain this component. 1: Tablets/Laptops/Consoles, 2: Laptops/Consoles, 3: Consoles only
+	var/hardware_size = 1				// Limits which devices can contain this component. 1: Tablets/Wristbound/Laptops/Consoles/Telescreen, 2: Laptops/Consoles/Telescreen, 3: Consoles/Telescreen
```

Comment:
Instead of having this stupid dumb comment here, it should just be flags. And if it is, this comment shouldn't exist.